src/pathman_workers.c

```diff
@@ -483,7 +483,7 @@ bgw_main_concurrent_part(Datum main_arg)
 
 		Oid		types[2]	= { OIDOID,				INT4OID };
 		Datum	vals[2]		= { part_slot->relid,	part_slot->batch_size };
-		bool	nulls[2]	= { false,				false };
+		char	nulls[2]	= { false,				false };
```

Comment:
Nice catch, but this is also wrong. SPI requires nulls to be equal to `'n'`. I guess we should get rid of `nulls` array and just pass `NULL` instead.

Reply:
Indeed, fixed.